install/Knative-with-ICP.md

```diff
@@ -0,0 +1,170 @@
+# Knative Install on IBM Cloud Private
+
+This guide walks you through the installation of the latest version of
+[Knative Serving](https://github.com/knative/serving) using pre-built images and
+demonstrates creating and deploying an image of a sample `hello world` app onto
+the newly created Knative cluster.
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+Knative requires an [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster v3.1.1. See [Installing IBM Cloud Private Cloud Native, Enterprise, and Community editions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html) in the IBM Knowledge Center for install instructions.
+
+1: [Install Docker for your boot node only](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html#prep_boot)
```

Comment:
Change to typical ordered list format 1: --> 1.

Reply:
Done